stylesheet.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.top-menu {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  padding-right: 1rem;
+}
+
+.top-menu li {
+  padding: 0.5rem;
+}
+
+main > section {
+  /* Or you could say main > *   you can remove these comments after reading */
+  text-align: center;
+  padding: 1rem 2rem;
+}
+
+.date {
+  text-align: right;
+  padding: 0 1rem;
+}
+
+.table {
+  border: 3px solid;
+}
+
+button {
+  box-shadow: 3px 4px #888;
+  width: 10vw;
+}
+
+.form input {
+  width: 245.77px;
+}
+
+.for-contact {
+  width: 80vw;
+  margin: auto;
+}
+
+.for-contact ul {
+  width: 70%;
+  margin: auto;
+  text-align: justify;
+}
+
+.a-border {
+  border-style: solid;
+  border-width: 2px;
+  border-radius: 25px 25px 55px 5px/5px 55px 25px 25px;
+}
+
+footer {
+  margin: 1rem;
+}
```

Comment:
Kindly improve upon the following aspects of your design:
- [ ] The footer should always stick to the bottom of the page
- [ ] The add button should align to the right within the form
- [ ] OPTIONAL
We want to enhance the user experience of this application, and therefore kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to mimic the provided wireframe template.

**Your design:**            |  **The original design:**
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/12745474/207770149-32cf3037-d68c-48e2-b0e7-79d979d11379.png) |  ![image](https://user-images.githubusercontent.com/12745474/143277255-658fcad7-519b-445d-b039-21a2187c6968.png)